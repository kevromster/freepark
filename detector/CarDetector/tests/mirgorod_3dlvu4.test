config = camera_nsk_mirgorod_3dlvu.txt
input = input/mirgorod_3dlvu4.png
expected = expected/mirgorod_3dlvu4.out
