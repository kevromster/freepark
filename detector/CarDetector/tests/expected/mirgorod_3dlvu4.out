[
parking id: 24
spots count: 1

{ lat: 54.853732; lon: 83.040122; complexity: NEWBIE }
]

[
parking id: 25
spots count: 0
]

[
parking id: 26
spots count: 1

{ lat: 54.853695; lon: 83.040088; complexity: NEWBIE }
]

